.ci/travis/travis.sh

```diff
@@ -39,12 +39,12 @@ test-fr)
 
 test-zh)
```

Comment:
shouldn't this change to cn?

Reply:
no, CN is country code. Only language code is important for message files (zh in this case).